css/styles.css

```diff
@@ -194,93 +194,41 @@
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
-  font-family: inter, serif;
+  font-family: 'Inter', serif;
   margin-right: 10px;
   margin-top: 10px;
 }
 
 .see-prjct-btn button {
-  background-color: #ff6b00;
+  background: #f55800;
```

Comment:
[OPTIONAL] I am sorry the previous review was not clear enough and didn't point where the problem was! :pray:   Great work trying to fix the contrast! Sadly the color change was not enough!. :disappointed: 

- [ ] Would you kindly fix the colors so they pass all the accessibility tests!
The contrast should have a ratio of 4.5:1, and as you can see below it doesn't meet the test requirements. :arrow_down: 
![image](https://user-images.githubusercontent.com/28903404/141035847-f830f37c-f070-4f61-b302-3439f9b9bf89.png)

You can quickly test contrast values by using [contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/contrast())
## Check it out!
![contrastChange](https://user-images.githubusercontent.com/28903404/141036860-58ad6154-3812-48af-b496-8de0205a8fc4.gif)